cmd/kubectl-direct_csi/drives_list.go

```diff
@@ -167,6 +167,10 @@ func listDrives(ctx context.Context, args []string) error {
 
 		msg := ""
 		for _, c := range d.Status.Conditions {
+			if string(d.Status.DriveStatus) == string(directcsi.DriveStatusReleased) {
```

Comment:
We don't need to convert to `string`,  we can directly check the field value. 
```suggestion
			if d.Status.DriveStatus == directcsi.DriveStatusReleased {
```